pkg/deploy/generator/resources_dev.go

```diff
@@ -380,24 +382,10 @@ fi
 EOF
 chmod +x /usr/local/bin/fix-podman-pause.sh
 
-# HACK - /tmp will fill up causing build failures
-# delete anything not accessed within 2 days
-cat >/usr/local/bin/clean-tmp.sh <<'EOF'
-#!/bin/bash
-
-find /tmp -type f \( ! -user root \) -atime +2 -delete
```

Comment:
If I'm reading this correctly, it looks like the replaced command `/sbin/tmpwatch 24h /tmp` will delete additional files (e.g. root-owned files). Is this intended/ok?

Reply:
If they're not accessed within 24 hours yes it will delete them.  This is fine as on RHEL8 `/tmp` is cleaned up on reboot anyway so all files are removed.